src/constants.iced

```diff
@@ -165,6 +165,8 @@ exports.constants = constants =
     EMPTY_JSON:        270
     BAD_API_URL:       280
     API_URL_MISMATCH:  281
+    UNEXPECTED_JSON:   282
+    CONTANT_MALFORMED: 283
```

Comment:
These code set also appears in client protocol/avdl/keybase1/prove_common.avdl

Reply:
crap. should i remove these new codes then?i thought it was internal only